
We must inherit the build environment to have the PATH to the wrappers as well
as their configuration.

  - Rene Rebe <rene@exactcode.de>

--- scons-0.96.1/engine/SCons/Action.py~	2004-08-23 16:56:03.000000000 +0200
+++ scons-0.96.1/engine/SCons/Action.py	2004-12-31 14:55:19.648282440 +0100
@@ -343,7 +343,8 @@
         for cmd_line in cmd_list:
             if len(cmd_line):
                 try:
-                    ENV = env['ENV']
+                    ENV = os.environ
+                    #ENV += env['ENV']
                 except KeyError:
                     global default_ENV
                     if not default_ENV:
